runelite-api/src/main/java/net/runelite/api/Experience.java

```diff
@@ -42,6 +42,11 @@
 	public static final int MAX_VIRT_LEVEL = 126;
 	public static final int MAX_SKILL_XP = 200_000_000;
 
+	/**
+	 * Maximum effective skill level at 13,034,431 experience.
+	 */
+	public static final int MAX_REAL_LEVEL = 99;
```

Comment:
I'd actually place this at the beginning of the class, above the virtual level/xp definitions.